server/src/__tests__/fixtures/directive.bb

```diff
@@ -8,4 +8,7 @@ FOO = '${APPEND}'
 SYMBOL_IN_STRING = 'hover is a package ${FOO} \
         parentFolder/hover should also be seen as symbol \
         this hover too, other words should not. \
-        '
\ No newline at end of file
+        '
+
+SRC_URI = 'file://foo.inc \
+        file://foo.inc'
```

Comment:
Weird for a bitbake file to provide the same URI twice. Not sure it would be compile through bitbake.